addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -659,6 +662,46 @@ public int getWascId() {
         return 33;
     }
 
+    private AlertBuilder createAlert(
+            int confidence,
+            String param,
+            String attack,
+            String evidence,
+            String otherInfo,
+            String alertRef) {
+        return newAlert()
+                .setConfidence(confidence)
+                .setParam(param)
+                .setAttack(attack)
+                .setEvidence(evidence)
+                .setOtherInfo(otherInfo)
+                .setAlertRef(getId() + alertRef);
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        List<Alert> alerts = new ArrayList<>();
+        alerts.add(
+                createAlert(
```

Comment:
More should be extracted, we should only need to pass what's different.

Reply:
What am I missing, it's all different?

Reply:
Confidence, other info, and alert ref are duplicated here and in the `scan` method.

Reply:
Tks will review.